libs/sdk-ui/src/base/vis/VisualizationProps.ts

```diff
@@ -46,6 +46,11 @@ export interface IVisualizationProps {
      * React component to display while loading data from the backend.
      */
     LoadingComponent?: React.ComponentType<ILoadingProps>;
+
+    /**
+     * Check if user is able to customize visualization height.
+     */
+    enableCompactSize?: boolean;
```

Comment:
is it needed also in this interface?  If yes it should have better comment, eg : "Enables more compact rendering of visualization in small containers/of small size"

Reply:
Is needed for the `CoreHeadline` props. Added new comment.

Reply:
This is most likely wrong. A prop that only makes sense for headline has no place in this common interface. Please consider moving it "closer" to headline components.